detail.py

```diff
@@ -0,0 +1,34 @@
+from django.conf import settings
+from django_mako_plus import view_function, jscontext
+from datetime import datetime, timezone
+from catalog import models as cmod
+import math
+
+@view_function
+def fix_html(x):
+    x = x.replace("&","&amp;")
+    x = x.replace("<","&lt;")
+    x = x.replace(">","&gt;")
+    return x
+
+@view_function
+def process_request(request, prod:cmod.Product=None):
+
+    product_id = prod.id
```

Comment:
Did you need this line? I thought you can just use prod.id if you just pass prod.

Reply:
Oh, you're right, thanks for the suggestion!